lib/ex_sdp/attribute/fmtp.ex

```diff
@@ -2,10 +2,19 @@ defmodule ExSDP.Attribute.FMTP do
   @moduledoc """
   This module represents fmtp (RFC 5576).
 
-  Parameters for H264 (not all, RFC 6184), VP8, VP9 and OPUS (RFC 7587) are currently supported.
+  Parameters for:
+  * H264 (not all, RFC 6184),
+  * VP8, VP9, OPUS (RFC 7587)
+  * RTX (RFC 4588)
+  * FLEXFEC (RFC 8627)
+  * Telephone Events (RFC 4733)
+  * RED (RFC 2198)
```

Comment:
RED has an additional parameter `forwardshift` described in [RFC 6354](https://www.rfc-editor.org/rfc/pdfrfc/rfc6354.txt.pdf). Shouldn't we add it also, as we adding support for RED? The other way change the description to look the same as for H264.

Reply:
As far as I understand this, `fwdred` is another Media Type so it is something equivalent to `red` -- it is not parameter for `red`.